Aptoide/appcoins-wallet-android
# [APPC-1168] Added ability to copy and past codes in wallet validation

**What does this PR do?**

   This PR adds the ability to copy and past numbers in the sms code screen in the wallet validation flow

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] CodeValidationFragment.kt

**How should this be manually tested?**

Create a new wallet and copy some text(numbers) . When prompt for the sms code paste the code. It should copy all fields 

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1168](https://aptoide.atlassian.net/browse/APPC-1168)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1168](https://aptoide.atlassian.net/browse/APPC-1168)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
